packages/env-options/src/env-options.js

```diff
@@ -78,15 +88,49 @@ export const makeEnvironmentCaptor = aGlobal => {
         setting = optionValue;
       }
     }
+    optOtherNames === undefined ||
+      setting === defaultSetting ||
+      arrayIncludes(optOtherNames, setting) ||
+      Fail`Unrecognized ${q(optionName)} value ${q(
+        setting,
+      )}. Expected one of ${q([defaultSetting, ...optOtherNames])}`;
     return setting;
   };
   freeze(getEnvironmentOption);
 
+  /**
+   * @param {string} optionName
+   * @returns {string[]}
+   */
+  const getEnvironmentOptionsList = optionName => {
+    const option = getEnvironmentOption(optionName, '');
+    return option === '' ? [] : stringSplit(option, ',');
```

Comment:
I prefer (but do not require) the equivalent:

```suggestion
    return stringSplit(option, ',');
```

Reply:
`stringSplit(option, ',')` would not be equivalent because `''.split(',')` is `['']` rather than `[]`. But we probably _should_ be freezing the output.
```suggestion
    return freeze(option === '' ? [] : stringSplit(option, ','));
```

Reply:
Good point about freezing the output. Done.